tools/src/main/java/com/nedap/archie/rmobjectvalidator/validations/RMPrimitiveObjectValidation.java

```diff
@@ -33,19 +33,26 @@
         }
         return result;
     }
-
     private static RMObjectValidationMessage createValidationMessage(Object value, String pathSoFar, CPrimitiveObject cobject) {
         List constraint = cobject.getConstraint();
         String message;
 
         if(constraint.size() == 1) {
             String constraintStr = ConstraintToStringUtil.constraintElementToString(constraint.get(0));
-            message = RMObjectValidationMessageIds.rm_INVALID_FOR_CONSTRAINT.getMessage(value.toString(), constraintStr);
+            message = RMObjectValidationMessageIds.rm_INVALID_FOR_CONSTRAINT.getMessage(getValueString(value), constraintStr);
         } else {
             String constraintStr = ConstraintToStringUtil.constraintListToString(constraint);
-            message = RMObjectValidationMessageIds.rm_INVALID_FOR_CONSTRAINT_MULTIPLE.getMessage(value.toString()) + "\n" +
+            message = RMObjectValidationMessageIds.rm_INVALID_FOR_CONSTRAINT_MULTIPLE.getMessage(getValueString(value)) + "\n" +
                     constraintStr;
         }
         return new RMObjectValidationMessage(cobject, pathSoFar, message);
     }
+
+    private static String getValueString(Object value) {
+        if(value == null) {
+            return "\"\"";
```

Comment:
This implies the value is an empty string (and not `null`). I can imagine this could cause some headache during debugging. Maybe change this to the word _empty_?